moabb/evaluations/metasplitters.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+from sklearn.model_selection import (
+    BaseCrossValidator,
+    StratifiedShuffleSplit,
+)
+
+
+class OfflineSplit(BaseCrossValidator):
+
+    def __init__(self, n_folds):
```

Comment:
In the documentation, include a way to use... Like this example:

https://pyhealth.readthedocs.io/en/latest/_modules/pyhealth/tasks/sleep_staging.html#sleep_staging_isruc_fn